### ISSUE TITLE
If you press the zero on the number pad and open a new tab, Windows Terminal crashes

### ISSUE THREAD

**Issue by Qantex** [microsoft/terminal#5293]
# Environment

```none
Windows build number: 10.0.18363.752
Windows Terminal version (if applicable): 0.10.781.0

Any other software?
```

# Steps to reproduce

- Start Windows Terminal
- Hold down the zero from the number pad
<img src="https://user-images.githubusercontent.com/47790579/78903215-1cbdfa80-7a7b-11ea-8476-c9fdd53136cc.png" width="180">

- Now try to open 5 to 10 new tabs by pressing the plus sign "+" with the mouse

# Expected behavior

- Windows Terminal should not crash

# Actual behavior

- Windows Terminal crashes

**Comment by zadjii-msft** [comment]
Does this happen with _any_ key, or is it somehow specific to numpad 0?

**Comment by jdebp** [comment]
I've managed to reproduce it with several keys, including `d`, the calculator keypad delete key, the spacebar, TAB, and F4.

**Comment by Qantex** [comment]
@DHowett If the **1** and also the **2** on the number pad is pressed, Windows Terminal crashes.

**Comment by zadjii-msft** [comment]
You know I bet this is the same crash that @DHowett-MSFT fixed in #5286, I'll let him confirm

**Comment by Qantex** [comment]
@zadjii-msft **Thank you** for informing @DHowett .

**Comment by Qantex** [comment]
@jdebp **Thank you very much** for **helping to solve the problem faster**.

**Comment by DHowett-MSFT** [comment]
Thank you for confirming every individual key that reproduces this problem. It will actually happen for all keys on the keyboard, indiscriminately.

This was caused by #4613, which was fixed in #5051.

Debugging information:

we try to clear the active selection, but _terminal is not yet initialized.

```
0:000> dx this->_terminal
this->_terminal                 : empty
                                  ^^^^^
                                  not supposed to be empty!
```

/dup #4613
